src/components/slider/GradientSlider.tsx

```diff
@@ -94,25 +94,31 @@ const GradientSlider = (props: Props) => {
     return color || sliderContext.value;
   }, [color, sliderContext.value]);
 
-  const renderDefaultGradient = useCallback(() => {
+  const getHueColor = useCallback(() => {
```

Comment:
It feels like this should be a memoize value more than a callback.. WDYT?

Reply:
Yes. Good idea